src/app.ts

```diff
@@ -1,8 +1,13 @@
 import express from 'express'
+import path from 'path'
 
 const app = express()
-const port = 3000
+const port = process.env.PORT || 3000
 
 app.get('/', (req, res) => res.send('Hello World!'))
 
+app.get('/admin/serviceHostKeys', (req, res) => {
+  res.sendFile(`${path.join(__dirname, '../../src/public/admin.html')}`)
```

Comment:
`res.render('admin')`. Put your admin.html inside the `views` folder. (this PR might have to be built on top of the views PR.